app/controllers/sign_up/completions_controller.rb

```diff
@@ -132,6 +138,7 @@ def pii_to_displayable_attributes
         birthdate: dob,
         phone: PhoneFormatter.format(pii[:phone].to_s),
         email: email,
+        verified_at: verified_at,
```

Comment:
@jmhooper @stevegsa  Ditto here, re: testing. There's not much low-level coverage of this controller that checks the attributes it passes to its view object, but the end result is verified through acceptance specs. Is that enough

Reply:
I think that if there is feature spec coverage then we are good